### ISSUE TITLE
offer more filetypes?

### ISSUE THREAD

**Issue by mastef** [makinacorpus/convertit#37]
Since unoconv can handle a lot of conversions - e.g. ppt -> pdf or pptx -> pdf ( from http://dag.wiee.rs/home-made/unoconv/ ), any reason to not include them from the get-go?

**Comment by leplatrem** [comment]
No, we didn't avoid those at all. It's just that this tool was driven by our use-cases in Geotrek.
Feel free to contribute this enhancement, add yourself to authors, and we'll happy to review it and merge !
